Makefile

```diff
@@ -1,5 +1,10 @@
 .PHONY: start start_dev
 
+IMAGE_TAG ?= local
+
+build:
+	docker build --no-cache -t editor-pdf-service:${IMAGE_TAG} .
```

Comment:
Not major, but you probably don't need the `--no-cache` option here. It's not going to cause any harm, but you might save a copy of seconds from your local builds.